src/deployment/flavors.md

```diff
@@ -3,41 +3,158 @@ title: Creating flavors for Flutter
 short-title: Flavors
 description: How to create build flavors specific to different release types or development environments.
 ---
+## What are flavors
+Have you ever wondered how to set up different environments in your Flutter app?
+Flavors (known as build configurations in iOS), create separate environments 
+for your app using the same code base. 
+You can use flavors to supply your app with different development versions.
 
-{% comment %}
-Eventually, we should actually include instructions,
-but by platform and we can then move the platform-specific
-instructions to /development/platform-integration/macos
-or ios or android, etc.
-sz: Yes, I've already asked Shamira (who will be working
-on the new docs), to plan for multiple pages.
-{% endcomment %}
-
-Do you need to set up product flavors for different development
-environments or release types?
-The community has written some articles and packages you might find useful.
-These articles address flavors for both iOS and Android.
-
-* [Creating flavors of a Flutter app][]
-* [Flavoring Flutter][]
-* [Flutter Ready to Go][]
-* [Build flavors in Flutter (Android and iOS)
-  with different Firebase projects per flavor][]
-* [Flutter 1.17 — no more Flavors, no more iOS Schemas.
-  Command argument that changes everything][]
-* [Multi-environment Flutter Projects with Flavors][]
-  (including support for web, Firebase and FlutterFire CLI)
-
-The following packages are listed alphabetically:
+Let’s say, you want to make a free and paid version of your Flutter app. 
+You can use flavors to set up both app versions 
+without writing two separate apps. 
+The free version of the app has basic functionality. 
+In contrast, the paid version has basic app functionality, extra features, 
+and different styles for paid users. 
+
+Another example where you could use flavors is for feature development. 
+If you’ve built a new feature and want to try it out, 
+you could set up a flavor and deploy that feature to a different API endpoint 
+than production. Your production code remains unaffected 
+until you are ready to deploy your new feature to production.
+
+Flavors function by letting you define compile time configurations 
+and set parameters that are read at runtime depending on your app needs.
+Setting up flavors is straightforward in Android, but requires more in-depth
+set up in iOS through Xcode. 
+
+This document guides you through setting up Flutter flavors for iOS and Android. 
+
+## Environment set up
+Prerequisites:
+* Xcode installed
```

Comment:
Xcode is required for configuring Flavors for iOS.

Android Studio isn't strictly required for configuring Flavors for Android, but presumably is required for building an APK?